pkg/translator/zipkin/zipkinv2/to_translator.go

```diff
@@ -437,9 +437,9 @@ func setTimestampsV2(zspan *zipkinmodel.SpanModel, dest ptrace.Span, destAttrs p
 	}
 }
 
-// unmarshalJSON inflates trace id from hex string, possibly enclosed in quotes.
+// UnmarshalJSON inflates trace id from hex string, possibly enclosed in quotes.
 // TODO: Find a way to avoid this duplicate code. Consider to expose this in pdata.
-func unmarshalJSON(dst []byte, src []byte) error {
+func UnmarshalJSON(dst []byte, src []byte) error {
```

Comment:
As the TODO comment states, this code is duplicated from internal core pdata, see here https://github.com/open-telemetry/opentelemetry-collector/blob/v0.119.0/pdata/internal/data/bytesid.go. As such, I'm not sure it's a good idea to expose this function from this package (certainly not with this name, as it does not at all give a good idea of what's it's doing). I think that ideally this function would be exposed from pdata.

Reply:
You might not even need this after all, if you use the suggestion I made on the ddreceiver component itself.

Reply:
That worked; thanks!